embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/internal/logs/EmbraceLogRecordExporterTest.kt

```diff
@@ -0,0 +1,22 @@
+package io.embrace.android.embracesdk.internal.logs
+
+import io.embrace.android.embracesdk.fakes.FakeLogRecordData
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.Test
+
+internal class EmbraceLogRecordExporterTest {
+
+    @Test
+    fun `export() should store logs in LogSink`() {
+        val logSink: LogSink = mockk(relaxed = true)
```

Comment:
Can this be replaced with a fake instead?

Reply:
Replaced.